src/utils/vc/credentials.js

```diff
@@ -293,7 +309,8 @@ export async function verifyCredential(
     documentLoader: docLoader,
   });
 
-  if (!skipSchemaCheck) {
+  const isAnoncredsDerived = isAnoncredsProofType(credential);
+  if (!skipSchemaCheck && !isAnoncredsDerived) {
```

Comment:
A comment explaining here the reason why `isAnoncredsDerived` is used to skip the check would be helpful.

And can you add a test for this?